README.md

```diff
@@ -239,6 +239,10 @@ Tarantool. In the above example, it is set to "mysecretpassword".
 Optional. Specifying this variable will tell Tarantool to listen for
 incoming connections on a specific port. Default is 3301.
 
+### `TARANTOOL_DEFAULT_METRICS_PORT`
```

Comment:
It exposes metrics in prometheus specific format, so it feels reasonable to name it somehow with 'prometheus' word. Or provide additional parameter, which will define a metrics format (it is okay to support only prometheus for now).

Reply:
done!